### PR TITLE
Convert single backtick code sections to triple backtick code sections for Arabic Guide articles (13 of 15)

### PR DESCRIPTION
**Pull request by RandellDawson**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Related to issue #36102 

This is one of a series of 15 PRs (75 or less per PR) which replace all applicable single backtick code blocks with the triple backtick code blocks plus language postfix (where applicable) to correct the mistakes made by the scripts which made the original translations of the Arabic Guide articles.  

**NOTES:**
* The scope of review for this PR should not focus on indentation or other style issues with the code sections.  However, it could be that the original Guide article had the wrong language post-fix (i.e. `html` should have been `css`).  In those instances, it would be fine to request changes.

* I only included a maximum of 75 files in each PR to make it easier to review on GitHub.

### CODE REVIEW

File: guide/arabic/python/if-elif-else-statements/index.md [hunk 1]
```diff
@@ -6,66 +6,72 @@ localeTitle: إذا كانت بيانات Elif Else
...
- x = 34 
- if x %  2 == 0:  # this is how you create a comment and now, checking for even. 
-  if x > 10: 
-    print("This number is even and is greater than 10") 
-  else: 
-    print("This number is even, but not greater 10") 
- else: 
-  print ("The number is not even. So point checking further.") 
-` 
+```
```
Comment (moT01) [inline guide/arabic/python/if-elif-else-statements/index.md]:
```suggestion
```python
```

File: guide/arabic/python/if-elif-else-statements/index.md [hunk 2]
```diff
@@ -6,66 +6,72 @@ localeTitle: إذا كانت بيانات Elif Else
...
+python
```
Comment (moT01) [inline guide/arabic/python/if-elif-else-statements/index.md]:
```suggestion
```

File: guide/arabic/python/if-elif-else-statements/index.md [hunk 3]
```diff
@@ -75,16 +81,18 @@ _ملاحظة فقط الشرط الأول أن يقيم كما `true` سيتم
 
 يمكننا أيضًا استخدام عبارات if-else في وظائف python المضمنة يجب أن يتحقق المثال التالي إذا كان الرقم أكبر من أو يساوي 50 ، إذا كانت الإجابة نعم صحيح:
 
- `python 
- x = 89 
- is_greater = True if x >= 50 else False 
- 
- print(is_greater) 
-` 
+```
```
Comment (moT01) [inline guide/arabic/python/if-elif-else-statements/index.md]:
```suggestion
```python
```

File: guide/arabic/python/if-elif-else-statements/index.md [hunk 4]
```diff
@@ -75,16 +81,18 @@ _ملاحظة فقط الشرط الأول أن يقيم كما `true` سيتم
...
+python
```
Comment (moT01) [inline guide/arabic/python/if-elif-else-statements/index.md]:
```suggestion
```

**Review by moT01 — CHANGES_REQUESTED** [review]
strange that this happened for just these two

**Comment by RandellDawson** [comment]
@moT01 I made your suggested changes.

**Review by moT01 — APPROVED** [review]
LGTM @RandellDawson 🎉

**Pull request merged**